server/nodeServer.js

```diff
@@ -26,15 +33,18 @@ const server = http.createServer((req, res) => {
   if (method === 'GET' && url === `/restaurants/${urlSplit[2]}/reservations/${urlSplit[4]}`) {
```

Comment:
For the url split, Beth strongly recommended destructuring when I did mine.
It ended up looking like this:
const [base, restaurantName, menu, meal, tag] = url.split('/');
It can help make the code easier to read.

Reply:
Nice, yeah I will make that change. Thanks.

Reply:
Nice, thanks! Good suggestion. I'll see if I can make that change.